frontend/src/routes/Infrastructure/Clusters/ManagedClusters/CreateCluster/controlData/ControlDataHelpers.js

```diff
@@ -75,6 +75,14 @@ export const getWorkerName = (control) => {
     return `Worker pool ${grpNum + 1}`
 }
 
+export function getOSTNetworkingControlData() {
+    // Kuryr should only be available for Openstack
+    const networkData = JSON.parse(JSON.stringify(networkingControlData))
```

Comment:
Is there a reason you didn't use cloneDeep here?

Reply:
Forgot about lodash!